companies/holvi.json

```diff
@@ -0,0 +1,27 @@
+{
+    "slug": "holvi",
+    "relevant-countries": [
```

Comment:
When trying to [register an account](https://my.app.holvi.com/register?country=en), a few more countries are listed as options. Please also include those.

Reply:
Hi @baltpeter, I've included the relevant countries based on what the user is able to select upon account sign up.